migrations/mystery_items.js

```diff
@@ -27,6 +27,8 @@ if (_id) {
   db.users.update({_id:_id}, update);
 } else {
   // multiple (once @ start of event)
-  db.users.update({'purchased.plan.customerId':{$ne:null}}, update, {multi:true});
+  db.users.find({'purchased.plan.dateTerminated':{$exists:true}}).forEach(function(user) {
```

Comment:
Yes, multiple docs at once is still needed.

However, I'm sorry, I wasn't clear before when I spoke about the $exists check for the case where the user's document does not contain the purchased.plan.dateTerminated field.

We need to find any user that has a value for 'purchased.plan.customerId' (i.e., they've bought a subscription at some time in the past) and whose subscription has not terminated. To determine if it has not terminated, you need to find any user where any one of the following is true:

- the purchased.plan.dateTerminated field does not exist
- the purchased.plan.dateTerminated field has a null value
- the purchased.plan.dateTerminated field has a date in the future

So the three dateTerminated checks need to be in the same $or clause. Let me know if that doesn't make sense! :)